docs/entities/serviceclaim.md

```diff
@@ -0,0 +1,48 @@
+# ServiceClaim
+
+A Service Claim represents a claim for service.
+
+## Specification
+
+The definition of a ServiceClaim can be obtained directly from our [ServiceClaim
+CRD](../../config/crd/bases/primaza.io_serviceclaims.yaml). The specification
+contains two required properties:
+
+- ServiceClassIdentity: A set of key/value pairs that identify the service
+  class. Examples of service class identity keys include type of service, and
+  provider of service. This property is required.
+- ServiceEndpointDefinitionKeys: An array of keys that is required for
+  connectivity. The values corresponding to each of these keys will be extracted
+  from the service. This property is required.
+
+There are other three optional fields. The EnvironmentTag and
+ApplicationClusterContext are mutually exclusive:
+
+- Application: Fields indentifies application resources through kind, apiVersion and label selector. 
+- EnvironmentTag: A string representing one of the environment.
+- ApplicationClusterContext:
+
+## Status
+
+The Status of the ServiceClaim is also defined under the [ServiceClaim
+CRD](../../config/crd/bases/primaza.io_serviceclaims.yaml). It contains only one
```

Comment:
I thought you have more properties to set the registered service resource the matched

Reply:
This is the only `required` field. I have mentioned others also.